src/python/foglamp/core/scheduler.py

```diff
@@ -404,14 +409,14 @@ def _schedule_next_timed_task(self, schedule, schedule_execution, current_dt):
             if current_dt.time() > schedule.time:
                 dt += self._ONE_DAY
 
-        # Advance to the correct day if specified
+        """Advance to the correct day if specified"""
         if schedule.day:
             while dt.isoweekday() != schedule.day:
                 dt += self._ONE_DAY
 
         schedule_execution.next_start_time = time.mktime(dt.timetuple())
 
-    def _schedule_first_task(self, schedule, current_time):
+    def _schedule_first_task(self, schedule: _Schedule, current_time: int) -> None:
```

Comment:
want to use type here only? what about other defs?!  if this is the plan we need to prepare the team to use it to keep things consistent.